### ISSUE TITLE
Add translation function or welcome message

### ISSUE THREAD

**Issue by alanspike02** [Rabek009/MoreGamemodes#1]
Hello, I'm a new member of the mod and I'm looking for a new mod. I am enjoying this mod.
I am a foreigner and when I host a room with this mod, most people leave the room because they can't understand the English.
So I am sorry to say this for very selfish reasons, but I would like to have a translation function or the ability to set a welcome message, like TOH if possible.
Thank you for your consideration.

**Comment by BornBasic** [comment]
i totally agree. the template could be very helpful. modes are awesome and can be improved much more
